src/components/__tests__/NumberSensorReadout.test.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
```

Comment:
The name of this file is inconsistent with the module it is testing

Reply:
Fixed.